frontend/.eslintrc.json

```diff
@@ -7,6 +7,7 @@
     }
   },
   "env": {
+    "jest": true,
```

Comment:
Since linting and formatting is exectured on tests also, you should fix syntax  (reason for the error `Forgot to run Prettier?` on CI)

Reply:
Are you talking about the failing CI action for the frontend linters, right? Or I missconfig [frontend/.eslintrc.json](https://github.com/intelowlproject/IntelOwl/pull/1460/files/83475678dfd91057959901c84e297478eded2815#diff-1ae7beaf1c104b27855f8dcc7060b74469bfa4f7c50e4a680cc0f36fc823b5b3) ?

Reply:
Yeah. Fixing syntax of tests to pass linters / formatters / CI